<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Graphics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="GridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="HorizontalLayout0_0">
        <item>
         <widget class="QLabel" name="AvailableFunctionsLabel">
          <property name="text">
           <string>Available functions: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="AvailableFunctionsComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="ThemesLabel">
          <property name="text">
           <string>Themes: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ThemesComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="AnimationsLabel">
          <property name="text">
           <string>Animations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="AnimationsComboBox"/>
        </item>
        <item>
         <spacer name="HorizontalSpacer0_0">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="HorizontalLayout1_0">
        <item>
         <widget class="QLabel" name="ALabel">
          <property name="text">
           <string>a = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ALineEdit">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="BLabel">
          <property name="text">
           <string>b = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BLineEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HorizontalSpacer1_0">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="DrawButton">
          <property name="text">
           <string>Draw</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
